src/components/Scorecard.vue

```diff
@@ -0,0 +1,430 @@
+<template>
```

Comment:
<img width="651" alt="Screenshot 2022-04-21 at 11 23 18 AM" src="https://user-images.githubusercontent.com/13185699/164383749-ed1d0782-3468-4ed6-bb9e-a20edb21d7a0.png">
handle the case when all questions are survey

Reply:
okayy